db/migrate/20230711112658_create_likes.rb

```diff
@@ -0,0 +1,12 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :post, null: false, foreign_key: true
+      t.references :user, null: false, foreign_key: true
+      t.datetime :created_at
+      t.datetime :updated_at
+
+      t.timestamps
+    end
```

Comment:
* Kindly ensure Table and column names should match the ERD diagram as stated in the [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/creating_data_model_project_v1_2.md#:~:text=Table%20and%20column%20names%20should%20match%20the%20ERD%20diagram.)